src/backend/tests/unit/test_views.py

```diff
@@ -447,6 +478,38 @@ def test_export_pdf(self, mock_auth, host_summaries, projects, currencies):
         response = client.post("/api/v1/report/pdf/?date_range=last_month")
         assert response.status_code == 200
 
+    @pytest.mark.parametrize('expected', [test_template_expected_data])
+    def test_templates(self, mock_auth, job_templates, expected):
+        client = APIClient()
+        response = client.get(f"/api/v1/templates/?page=1&page_size=10")
+        assert response.status_code == 200
+        data = response.json()
+        assert data == expected
+
+    @pytest.mark.parametrize('expected', [test_labels_expected_data])
+    def test_labels(self, mock_auth, labels, expected):
+        client = APIClient()
+        response = client.get(f"/api/v1/labels/?page=1&page_size=10")
+        assert response.status_code == 200
+        data = response.json()
+        assert data == expected
+
+    @pytest.mark.parametrize('expected', [test_organizations_expected_data])
+    def test_organizations(self, mock_auth, organizations, expected):
+        client = APIClient()
+        response = client.get(f"/api/v1/organizations/?page=1&page_size=10")
+        assert response.status_code == 200
+        data = response.json()
+        assert data == expected
+
+    @pytest.mark.parametrize('expected', [test_projects_expected_data])
+    def test_projects(self, mock_auth, projects, expected):
+        client = APIClient()
+        response = client.get(f"/api/v1/projects/?page=1&page_size=10")
+        assert response.status_code == 200
```

Comment:
Since you are already parameterizing these tests, we could combine all these tests into a single testing function.